Tests/TensorFlowTests/Helpers.swift

```diff
@@ -36,3 +36,13 @@ internal func assertEqual<T: TensorFlowFloatingPoint>(
 ) {
     assertEqual(x.scalars, y.scalars, accuracy: accuracy, message, file: file, line: line)
 }
+
+func withTensorLeakChecking(file: StaticString = #file,
```

Comment:
Should we follow the convention of formatting long argument lists as follows? It's pretty much done like that in most of the `swift-apis` repo.

```swift
func withTensorLeakChecking(
    file: StaticString = #file,
    line: UInt = #line,
    _ body: () throws -> Void
) rethrows {
```

Reply:
+1 for this helper and the memory leak tests btw :)